(doc) setup starter files, generate historical data

aploftus opened the pull request:
* Added starter files from thesis repo
* Set up basic express server
* Initialized basic redis-server client
* Added data generation files for users, tweets, feeds

It's a big commit, all for setup. Thank you in advance for a review!

https://trello.com/c/aq4rjeFS

aploftus committed:
(docs) Add starter files

aploftus committed:
(docs) Set up npm modules and eslint pre-commit hook

aploftus committed:
(doc) Add express server

aploftus committed:
(docs) Setup basic redis config

aploftus committed:
(feat) Generate redis seed files

rtschultz commented on data-generation/tweetGeneratorHelper.js:
```diff
@@ -0,0 +1,74 @@
+const fakeHex = () => {
+  let result = '';
+  for (let x = 0; x < 6; x++) {
+    let randIndex = Math.floor(Math.random() * 16);
```
Consider a helper function for generating random numbers (DRY)

rtschultz commented on data-generation/userGenerator.js:
```diff
@@ -0,0 +1,45 @@
+const jsonfile = require('jsonfile');
```
remove deps you aren't using.

aploftus committed:
(feat) Add redis client stream mass add option for users

Is faster than using the cli --pipe, but needs more memory, so it
kicks out early (and is an all-or-nothing). Needs monitoring.

aploftus committed:
(refactor) Generate CSV user and tweet data for Cassandra
